styles/website.css

```diff
@@ -1716,3 +1713,33 @@ img {
         display: none;
     }
 }
+.infobox {
+    display: inline-flex;
+    border-left: .4em solid;
+    border-top: .1em dashed;
+    border-bottom: .1em dashed;
+    border-bottom-left-radius: .2em;
+    border-top-left-radius: .2em;
+    border-bottom-right-radius: .2em;
+    border-top-right-radius: .2em;
+    border-color: inherit;
+    opacity: 80%;
+    border-right: .1em dashed;
+    padding: .5em .5em .5em 0;
+    background: 0 0;
+    width: 100%
+}
+.infobox p {
+    padding-top: .5em;
+    font-size: .95em;
+    margin-bottom: 0;
+    padding-bottom: .5em
+}
```

Comment:
If the `p` tag is just used for spacing, do we need it at all? Or, alternatively, if we do need the `p`, could we get rid of the `div`?

Reply:
I think the reason I split this while testing was the padding was messing up the location of the info symbol, I can test merging them together when I revisit this. If it doesn't work combined, some `!important`s for the `::before` might help. I *think* getting rid of the `div` will mean the border only surrounds the text and not the symbol, but I guess moving `.infobox::before` to `.infobox p::before` would fix that if that's even possible.

Reply:
Also, you might've noticed I split the border-radius command out because I was playing around with different border radius's on the left and different styles and wanted to keep it split until I was satisfied with the style. Right now they all have the same value so it's redundant, but just an fyi as to why those are split.

Reply:
Merged in the `p` tag in https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/pull/55/commits/76975702f2301f3c588cb4f66955a8b4b0f69b9c and got rid of all the redundant code.